core/wiki/macros/keyboard-driven-input.tid

```diff
@@ -0,0 +1,50 @@
+title: $:/core/macros/keyboard-driven-input
+tags: $:/tags/Macro
+
+\define get-filter-list()
+$(filter1)$ $(filter2)$
+\end
+
+\define input-next-actions(afterbefore:"after",reverse:"",tiddler,storeTitle,field,index,filter1,filter2,refreshTitle)
+<$list filter="[<__storeTitle__>get[text]minlength<__filterMinLength__>] [<__filterMinLength__>match[0]] +[limit[1]]" variable="ignore">
+<$vars userInput={{{ [<__storeTitle__>get[text]] }}} selectedItem={{{ [<selectionTiddler>get[text]] }}}>
+<$set name="filter1" filter=<<__filter1__>>>
+<$set name="filter2" filter=<<__filter2__>>>
+<$set name="filterListRaw" value=<<get-filter-list>>>
+<$set name="filterList" filter="[enlist:raw<filterListRaw>indexed[]]">
```

Comment:
Could we not skip the previous `set widget` for `filterListRaw` and just use the macro `get-filter-list` directly here?

`<$set name="filterList" filter="[enlist:raw<get-filter-list>indexed[]]">`

Reply:
Oh, I think you're right. I'll test and change it

Reply:
Thanks @saqimtiaz , I've changed it now